test/runtests.bat

```diff
@@ -7,7 +7,11 @@ REM compile the code into the bin folder
 javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
 
 REM run the program, feed commands from input.txt file and redirect the output to the actual.txt
-java -classpath ..\bin seedu.addressbook.AddressBook < input.txt > actual.txt
+java -classpath ..\bin seedu.addressbook.AddressBook %" " > actual.txt
```

Comment:
I have no idea about windows batch scripts at all, but what does putting the `%` in front of the `" "` actually do?

Also, does `< NUL` not work? (redirect input to the null device)